sandbox/id-repository-mz.properties

```diff
@@ -36,19 +36,19 @@ mosip.idrepo.objectstore.bucket-name=idrepo
 mosip.idrepo.objectstore.adapter-name=s3Adapter
 
 #IDRepo identity/vid service authentication details
-mosip.idrepo.client-id=mosip-idrepo-client
-mosip.idrepo.secret-key=9c8c80c7-3a28-4df9-8b6f-153b241c4179
-mosip.idrepo.app-id=idrepo
+mosip.idrepo.auth.client-id=mosip-idrepo-client
+mosip.idrepo.auth.secret-key=9c8c80c7-3a28-4df9-8b6f-153b241c4179
```

Comment:
we should encrypt secrets and than update the same

Reply:
Fixed